src/main/resources/migrations.xml

```diff
@@ -4114,6 +4114,31 @@
 	</changeSet>
 
 	<!-- From this point on, mssql is no longer supported, so only postgresql migrations are needed -->
+	<changeSet id="add_organization_responsible_positions" author="nur.yilmaz">
+		<createTable tableName="organizationResponsiblePositions">
+			<column name="organizationUuid" type="${uuid_type}">
+				<constraints nullable="false"/>
+			</column>
+			<column name="positionUuid" type="${uuid_type}">
+				<constraints nullable="false"/>
+			</column>
+		</createTable>
+		<addUniqueConstraint tableName="organizationResponsiblePositions" columnNames="organizationUuid, positionUuid" constraintName="uniqueOrganizationResponsiblePositions" />
```

Comment:
At some point I tried to introduce the `UQ_` prefix for unique constraints, but it is far from consistent. :wink: